open-company/open-company-web
# Board creation

Card: https://trello.com/c/08kqHWiJ

To test:
- cleanup auth and storage DBs
- signup with email
- create an org
- visit a default board page and click the board settings icon
- do you see the slack mirror setup? you shouldn't
- signup with slack but do not connect the bot (deny second auth step on slack)
- create an org
- visit a default board page and click the board settings icon
- go to team settings (old UI) and add the bot to the slack org
- go back to the board page and open the board settings
- do you see the slack mirror setup now? you should!
- click create new board
- enter a name
- do you see the slack mirror setup? you shouldn't
- save the board
- open board settings modal
- do you see the slack mirror setup now? you should!
- try adding/removing the slack mirror and saving
- test comments mirroring for a random entry
